var/spack/repos/builtin/packages/py-proj/package.py

```diff
@@ -0,0 +1,28 @@
+from spack import *
+
+class PyProj(Package):
+    """Python interface to the PROJ.4 Library."""
+    homepage = "http://jswhit.github.io/pyproj/"
+    url      = "https://github.com/jswhit/pyproj/tarball/v1.9.5.1rel"
+
+
+    version('1.9.5.1', 'a4b80d7170fc82aee363d7f980279835')
+
+    # We need the benefits of this PR
+    # https://github.com/jswhit/pyproj/pull/54
+    version('citibeth-latlong2',
+        git='https://github.com/citibeth/pyproj.git',
+        branch='latlong2')
+
+    extends('python')
+
+    depends_on('py-cython')
+    depends_on('py-setuptools')
```

Comment:
Huh?
On May 31, 2016 11:14 AM, "Ben Boeckel" <notifications@github.com> wrote:

> In var/spack/repos/builtin/packages/py-proj/package.py:
>
> > +    homepage = "http://jswhit.github.io/pyproj/"
> > +    url      = "https://github.com/jswhit/pyproj/tarball/v1.9.5.1rel"
> > +
> > +
> > +    version('1.9.5.1', 'a4b80d7170fc82aee363d7f980279835')
> > +
> > +    # We need the benefits of this PR
> > +    # https://github.com/jswhit/pyproj/pull/54
> > +    version('citibeth-latlong2',
> > +        git='https://github.com/citibeth/pyproj.git',
> > +        branch='latlong2')
> > +
> > +    extends('python')
> > +
> > +    depends_on('py-cython')
> > +    depends_on('py-setuptools')
>
> @mathstuf <https://github.com/mathstuf>: have you used an external proj?
>
> Never used py-proj myself (I'm more familiar with proj4, but that's
> really just "it's in our build stack and I'm one of the build stack guys"
> kind of relationship); just pointing out that mixing multiple libraries
> with the same symbols tends to not go too well…
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/LLNL/spack/pull/717/files/552a2815cd82b04c74eac44352f5e812ea0fa7f6#r65201976>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AB1cd6AWPD01dI1JJSI3dq3c30cvNkq4ks5qHFA-gaJpZM4H9_nz>
> .
>